### PR TITLE
uwt.0.0.3 release

### PR DESCRIPTION
**Pull request by fdopen**
new uwt version

uwt provides OCaml bindings for libuv.
The main loop of libuv is integrated into lwt,
the light-weight cooperative threads library.

This is mostly a maintaince release:

- new functions that have recently been added to libuv are now
  supported

- the higher level interface that mimics modules from Lwt_unix
  incorporates the latest upstream changes

- a few minor bugs have been fixed

uwt is still in an early development stage

### CODE REVIEW

File: packages/uwt/uwt.0.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "lwt"
+  "ounit" {test & >= "2.0"}
+  "ppx_deriving" {test & >= "2.0"}
+  "ppx_import" {test & >= "1.0"}
+]
+depopts: [
+  "conf-libuv"
+]
+conflicts: [
+ "lwt" {< "2.4.7"}
```
Comment (OCamlPro-Henry) [inline packages/uwt/uwt.0.0.3/opam]: Was there a reason for you to move the constraint from `depends` to `conflict` ? If not, the solver better handle the former. Using `confilct` is only required for `depopts`.

Comment (fdopen) [inline packages/uwt/uwt.0.0.3/opam]: Thanks for the hint, I've changed it accordingly.
lwt was originally intended as depopts (libuv works synchronously, too), but then I've changed my mind and enforced the dependency on lwt.

Comment (OCamlPro-Henry) [inline packages/uwt/uwt.0.0.3/opam]: Make sense, thanks.

**Pull request merged**